recipes/bash_kernel/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "bash_kernel" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a08c84eddd8179de5234105821fd5cc210015671a0bd3cd0bc4f631c475e1670
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - {{ PYTHON }} -m flit install
+    - {{ PYTHON }} -m bash_kernel.install --sys-prefix
```

Comment:
```suggestion
    - {{ PYTHON }} -m bash_kernel.install --prefix {{ PREFIX }}
```

Reply:
I will have to change my sos-notebook recipe since it also [uses `--sys-prefix`](https://github.com/conda-forge/sos-notebook-feedstock/blob/master/recipe/meta.yaml#L17)